ONSdigital/sdc-int-rh-ui
# Add cookies and privacy pages

# Motivation and Context

We want to be able to give information to the users about cookies and privacy/data protection
# What has changed



Cookies page added
Privacy and data protection page added
# How to test?



Build and run the service off this branch
Check that the links to cookies and the privacy/data protection pages work and take you to the correct page.
Check that the info on these pages match up with what is linked to on the card.
# Links



https://trello.com/c/w0QdMA7E/152-create-cookie-and-privacy-pages-and-provide-eq-with-links-5
# Screenshots (if appropriate):
